### ISSUE TITLE
suggestion: always have the description come first in JSDocs for unstable symbols

### ISSUE THREAD

**Issue by iuioiua** [denoland/deno#25103]
A symbol being unstable is secondary compared to a description of what that symbol does. I suggest we move the description to be first, and any unstable notices second.

CC @thisisjofrank @crowlKats

![image](https://github.com/user-attachments/assets/726d2a09-4da4-4977-bc4c-fc9b394b03b0)

**Comment by dsherret** [comment]
I'm not sure about this. It being moved to the end increases the chance of it being cut off in auto-complete. Also, when someone is going to use something, they should see right away that it's unstable.

**Comment by crowlKats** [comment]
this is a bug with docs.deno.com postprocessing of typings, for which i have a local fix, which will land later this week

**Comment by iuioiua** [comment]
To be clear, I suggest moving the unstable notice to the 2nd paragraph, not the last paragraph of the symbol's documentation. I think it's more important to the user to know what the symbol does, over its instability status.

> this is a bug with docs.deno.com postprocessing of typings, for which i have a local fix, which will land later this week

Sorry, what exactly is a bug? Perhaps, we can have `@experimental` tags be somehow represented even in symbol lists like the screenshot I share.

**Comment by dsherret** [comment]
> To be clear, I suggest moving the unstable notice to the 2nd paragraph, not the last paragraph of the symbol's documentation. I think it's more important to the user to know what the symbol does, over its instability status.

I don't agree with that. People rarely even read docs and I think it should be the first thing people see in capital letters in order to catch their eye. This is especially important given we no longer require `--unstable` for type checking.

**Comment by iuioiua** [comment]
Let's agree to disagree then 😛

Either way, perhaps the suggestion in my previous comment might satisfy both preferences. Maybe it could be an "unstable" button or similar next to the symbol name.

**Comment by LeonxLJX** [comment]
I'd like to take this one.

Reading the issue, this looks like a documentation-clarity gap rather than a code defect: the current page doesn't give readers enough to self-diagnose, which is why it surfaces here as an issue instead of being answered by the docs.

My plan: locate the relevant section, confirm the described behavior against the current source (so the docs match what the code actually does today, not what it did when the page was written), then tighten the wording and add the missing piece. I'll also scan neighboring pages for the same inconsistency, otherwise the fix just relocates the confusion.

One thing I'd like to confirm before writing: would you prefer a minimal correction scoped to this page, or should I also cross-link it from the related guides if the same gap shows up there?

(claiming via @LeonxLJX)
